qiskit_nature/drivers/second_quantization/pyquanted/pyquantedriver.py

```diff
@@ -51,13 +50,12 @@
 
 logger = logging.getLogger(__name__)
 
-try:
+if _optionals.HAS_PYQUANTE2:
```

Comment:
I don't know if you mind, but this isn't a lazy test - the Boolean evaluation will be triggered as soon as this module is imported.

Reply:
Yes, I knew. We had those at the top because they are being used in different parts of the file and even in type annotations like:
```
 self._mol: pyquante_molecule = None
 self._bfs: basisset = None
``` 

I will see if it is possible to move them inside the methods.